### DISCUSSION TITLE
Timesheet export: user name in footer or behind title if printed by admin?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#5241] [Questions & Answers]
After studying https://github.com/kimai/kimai/blob/main/templates/export/pdf-layout.html.twig for hours, I am still not able to modify it  to display the user name when the timesheet is created by an admin.

{{ entry.user.displayName }} shows the selected user in the UserColumn.

However, {{ entry.user.displayName }} does not work elsewhere, probably because the admin could select more than one user.

Any idea how I could show the name of the user whose timesheet is created by an admin in the footer or after the title?

**Comment by xn1nja** [comment]
This is exactly what i was looking for too. Any update on this?

**Reply by kevinpapst** [reply]
This is exactly what these lines do:
https://github.com/kimai/kimai/blob/main/templates/export/pdf-layout.html.twig#L15-L19

**Reply by EduNetEurope** [reply]
Thanks, @kevinpapst.
Don't these lines show a "column" rather than display the name in either the footer or title?
We tried using {{ entry.user.displayName }} in the title and the footer, but it will not display the user name. It works only in the column.
